.github/workflows/translation-check.yaml

```diff
@@ -9,14 +9,18 @@ name: translation file checks
 jobs:
     translation-check:
         runs-on: ubuntu-latest
-        container:
-            image: archlinux:latest
```

Comment:
I don't think this is desirable.
I get that it might look strange that we're running arch in ubuntu, but it's nice to test and run all code related things on a arch machine - to catch breaking changes in the OS.

Reply:
I totally understand what you're thinking, but I found it impossible to run `git diff --name-only` in a docker environment

Reply:
I droped my changes of `translation-check.yaml`, this PR only updates the po file.

Reply:
The main issue regarding the check appers to be the path. I'll fix it in another PR :)